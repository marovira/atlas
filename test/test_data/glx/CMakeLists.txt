set(ATLAS_GLX_TEST_DATA_ROOT ${ATLAS_TEST_DATA_ROOT}/glx)

set(ATLAS_GLX_TEST_DATA_LIST
    ${ATLAS_GLX_TEST_DATA_ROOT}/glx_empty_file.glsl
    ${ATLAS_GLX_TEST_DATA_ROOT}/glx_single_line.glsl
    ${ATLAS_GLX_TEST_DATA_ROOT}/glx_simple_file.glsl
    ${ATLAS_GLX_TEST_DATA_ROOT}/glx_simple_file_comments.glsl
    ${ATLAS_GLX_TEST_DATA_ROOT}/glx_single_include.glsl
    ${ATLAS_GLX_TEST_DATA_ROOT}/glx_multiple_includes.glsl
    ${ATLAS_GLX_TEST_DATA_ROOT}/glx_nested_include.glsl
    ${ATLAS_GLX_TEST_DATA_ROOT}/glx_circular_include.glsl
    ${ATLAS_GLX_TEST_DATA_ROOT}/glx_header_comments.glsl
    ${ATLAS_GLX_TEST_DATA_ROOT}/glx_c_comments.glsl

    ${ATLAS_GLX_TEST_DATA_ROOT}/nested_include.glsl
    ${ATLAS_GLX_TEST_DATA_ROOT}/uniform_bindings.glsl
    ${ATLAS_GLX_TEST_DATA_ROOT}/uniform_matrices.glsl
    ${ATLAS_GLX_TEST_DATA_ROOT}/circular_include_a.glsl
    ${ATLAS_GLX_TEST_DATA_ROOT}/circular_include_b.glsl
    PARENT_SCOPE)
