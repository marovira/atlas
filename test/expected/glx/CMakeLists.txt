set(ATLAS_GLX_EXPECTED_ROOT ${ATLAS_EXPECTED_ROOT}/glx)

set(ATLAS_EXPECTED_GLX_LIST
    ${ATLAS_GLX_EXPECTED_ROOT}/glx_single_line.expected
    ${ATLAS_GLX_EXPECTED_ROOT}/glx_simple_file.expected
    ${ATLAS_GLX_EXPECTED_ROOT}/glx_simple_file_comments.expected
    ${ATLAS_GLX_EXPECTED_ROOT}/glx_single_include.expected
    ${ATLAS_GLX_EXPECTED_ROOT}/glx_multiple_includes.expected
    ${ATLAS_GLX_EXPECTED_ROOT}/glx_nested_include.expected
    ${ATLAS_GLX_EXPECTED_ROOT}/glx_circular_include.expected
    ${ATLAS_GLX_EXPECTED_ROOT}/glx_header_comments.expected
    ${ATLAS_GLX_EXPECTED_ROOT}/glx_c_comments.expected
    PARENT_SCOPE)
